### ISSUE TITLE
The badges don't render nicely and are confusing / redundant

### ISSUE THREAD

**Issue by constantinpape** [bioimage-io/archive.bioimage.io#55]
The badges we currently have are either very technical (e.g. License) or redundant ("powered by ImJoy").
If we add more it will also get very crowded; the rendering is not very nice for the models with more than 1 badge already.

Should we just leave this out?

**Comment by oeway** [comment]
The purpose of badges is to provide concise metrics, link, extra information for items, it has been quite successfully used for READMEs of github repos.

These are some use cases of badges: 
 * There are badges such as "Open in Colab" and "Launch Binder" has already been used widely, keeping those would be very natural to guide users to external notebooks.

 * Some items contains numbers of elements, say, a models contains 20 weights files, a dataset contains 80,000 images, these will be useful to shown as a badge to give a quick feeling about the item. 

 * In longer term, especially when we have many more models, we should provide certification service to highlight/award models that are popular, small, fast, high-quality, reproducible, low CO2 footprint etc.

 * For models or applications, we would like also to show whether they passed our test. When we upgrade the model description format, some model may be outdated, we want to show a "warning" badge for models that are outdated.

`badges` is a configurable field that can be specified by the model provider/community partner, I think we should give some guidelines but leave the freedom to the one who make the RDF to decide.

In general, I think we should have the badges, a badge/medal-like element to show concise information about the items, the current badges listed in the website are demos about badges ("powered by ImJoy" is just a demo that you add custom badges). Regarding the rendering, I don't have better ideas on how to improve it, suggestions are welcome.

A side note for the license, I don't think it's very technical, but quite relevant for the users. Although you see many of the example models has the same license such as MIT, for the weights, it is perhaps make more sense to use creative commons. And that will have more implications on how the users use the model, dataset, or notebooks. For example, CC BY 2.0 will allow you train on top of a weights file and share the further trained weights. If it's CC BY-ND 2.0, then you are not allowed to further share the fine-tuned weights.

**Comment by constantinpape** [comment]
> The purpose of badges is to provide concise metrics, link, extra information for items, it has been quite successfully used for READMEs of github repos.

Yes, they fit nicely in github readmes, but I think they make everything look very crowded here; we have far less space for a single model compared to a github start page.

> For example, CC BY 2.0 will allow you train on top of a weights file and share the further trained weights. If it's CC BY-ND 2.0, then you are not allowed to further share the fine-tuned weights.

Sure, license information is important, but I think it's fine to show it in the more detailed description; the main purpose of the landing page should be to connect users with the models for their problem in the tool they want to use ;).

> Regarding the rendering, I don't have better ideas on how to improve it, suggestions are welcome.

Not sure we can improve this much; the space we have will always be limited, so if there are too many badges it will look crowded.
Only option I can think of is collapsing the badges as soon as there are more than the number that fits well (2 or 3) and then expanding when clicking on the collapsed badge symbol.
